### ISSUE TITLE
Duplicate teams, duplicate members

### ISSUE THREAD

**Issue by efg** [expertiza/expertiza#1500]
For Program 2 in CSC/ECE 517, Fall 2019 (assignment 991), there are many anomalies with teams.  At one point, the following anomalies were noted (though some people may have changed teams since then).

- Team_470 and Team_471 both contained the same members, apkulka2 and svshingt.
- Team_576 had admaycoc on it twice
- Team_587 had swen4 on it twice
- Team_2333 had jcai3 on it twice
- Team_590 also contained admaycoc, who was somehow on two teams at the same time, one of them twice.

This is clearly an error, because a student should not be able to join one team without leaving the team that (s)he is already on.

Here is the membership of the affected teams, at the point where this snapshot was taken.

Team_471	apkulka2	svshingt
Team_2333	dwang37	wfu4	jcai3	jcai3
Team_576	dmendez	admaycoc	admaycoc
Team_587	bzhang37	swen4	swen4
brainyHacks	magavade	ajain37	ajain37
Team_590	admaycoc

**Comment by efg** [comment]
Here are reports from the various students involved.

Aidan Maycock:
Initially, while searching for a team I went as far as selecting the "your team" link in the program 2 page. 
I had been assigned a team name, though there were out course no other members at this point. 

When I was invited to Daniel's team I believe the first time I tried to accept the process failed, before I refreshed and completed the process.

It also should be noted that Pranav was and still is listed twice on our program 1 team. I am not any sort of authority on this as prior to this semester I've never used expertiza. 

Ayush Jain:
We tried to create a team earlier but since it was causing an error. As soon as one member accept the invitation, it creates two entry in the team and then third member was not able accept the invitation. So i think, since i couldn't accept the invitation so created a new team. Probably the old team still has some data specific to me. 

I am the same student, who spoke to you in the last class regarding the bug. I'll soon raise the issue in github expertiza. 

Siwei Wen:
Sorry, I really do not know how it happened, and I only accepted the invitation once successfully. 

However, that problem is easy to be fixed for us. I quit my team and asked my teammates to invite me again, then there is only one me.

Jing Cai:
Yes, I think I accepted it twice. I accepted the invitation at first. However, when I logged in a few days after, I found a new invitation. As I almost forgot whether I accepted the invitation or not, I accepted it again. That's might cause this bug.
Hope that information helps.

Anuja Kulkarni:
Initially I was the only one assigned to Team_470. My project partners (Saurabh and Sameer) were not assigned to any. So I sent them an invitation and added them to Team_470. So currently our team name is Team_470.
If this helps you in any way: My third project partner, Sameer, accepted the invitation just right now.
I hope this helps you in debugging the problem.

Saurabh Shingte:
I'm on Team_470 for Program 2. I accepted an invite sent by Ms. Anuja for Team_470. Other than that I did not accept any other invitation nor I created any team.
